src/components/foodDescription/FoodDescription.js

```diff
@@ -24,7 +57,12 @@ class FoodDescription extends React.Component{
         return (
             <div className="food-description flex">
                 <Overview categories={Object.keys(this.state.foodListData)} />
-                <FoodList foodListData={this.state.foodListData} />
+                <FoodList foodListData={this.state.foodListData} handleAddToCart={this.handleAddToCart}/>
```

Comment:
This should be in the render fn

Reply:
it is in the render function only right?

Reply:
> it is in the render function only right?

No you're returning jsx from a lifecycle method???

Reply:
i think line 16 to 23 got collapsed so its causing confusion